mailpoet/assets/js/src/newsletters/send/sender_address_field.jsx

```diff
@@ -49,7 +49,7 @@ class SenderField extends Component {
           field={this.props.field}
           onValueChange={this.onChange}
         />
-        <div className="regular-text">
+        <div className="regular-text" style={{ marginTop: '2.5rem' }}>
```

Comment:
Is this margin necessary? It looks better without it to me.

Reply:
Yeah, but that's to fix this issue https://user-images.githubusercontent.com/30554163/176655015-777259ac-5748-48b0-8f27-41befac015a9.png